DESCRIPTION

```diff
@@ -79,7 +79,9 @@ Imports:
     crayon,
     rlang (>= 0.4.0),
     fastmap (>= 1.0.0),
-    withr
+    withr,
```

Comment:
If you wanted to keep the dependencies light, you could make put these in suggests. Then you'd only need to install if you wanted to use `markdown()`.

Reply:
@schloerke, @wch, and myself discussed adding the dependency more today, and also tossed around the idea of adding this feature to `htmltools` instead of to Shiny. We concluded Shiny is the right place and that Imports is preferable to Suggests.